File: go/take_derivative.go
package main

import (
	"fmt"
)

/*
func Derive(coefficient, exponent int) string {

	return strconv.Itoa(coefficient*exponent) + "x^" + strconv.Itoa(exponent-1)
}
*/

func Derive(coefficient, exponent int) string {
	return fmt.Sprintf("%dx^%d", coefficient*exponent, exponent-1)
}

func main() {
	fmt.Println(Derive(7, 8))
	fmt.Println(Derive(5, 9))
}

File: python/take_derivative.py
def derive(coefficient, exponent): 

    return str(coefficient*exponent) + "x^" +str(exponent-1)

# el mejor
def derive(coefficient, exponent): 
    return f'{coefficient * exponent}x^{exponent - 1}'

print(derive(7, 8))
print(derive(5, 9))